src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newArray = [];
+  const newState = { ...state };
```

Comment:
variable name should describe what is stored inside . Not the purpose you use it for, not it’s data type. So let’s avoid naming our variables like result , newArray or obj